amundsen_application/models/user.py

```diff
@@ -78,6 +81,9 @@ def preprocess_data(self, data: Dict) -> Dict:
             if app.config['GET_PROFILE_URL']:
                 data['profile_url'] = app.config['GET_PROFILE_URL'](data['user_id'])
 
+        # Fallback since search and metadata use a different key for 'full_name'
+        data['full_name'] = data.get('full_name', data.get('name'))
```

Comment:
Ideally, there should be a consistent data format among FE, Metadata, and Search.

Reply:
Done.